### ISSUE TITLE
rule based uploader does not scroll well/at all

### ISSUE THREAD

**Issue by martenson** [galaxyproject/galaxy#6100]
Note that columns go all the way to H but I cannot scroll horizontally. However there seems to be a dedicated space below the table for a scrollbar?

I can scroll vertically but there is also no scrollbar visible. No visible space for it this time though.

I am on Chrome MacOS with Apple touchpad.

ping @jmchilton 

![screenshot 2018-05-09 10 34 30](https://user-images.githubusercontent.com/1814954/39820778-97694986-5374-11e8-951a-7d2362aba1a3.png)

**Comment by jmchilton** [comment]
It scrolls for a chosen few, that is the will of the rule builder. You have not been chosen - this saddens me but that is the will of the builder.

**Comment by martenson** [comment]
@jmchilton reported by @nekrut 🙃

**Comment by martenson** [comment]
Moving this to 18.09 since it is a non-blocking corner case.

**Comment by jmchilton** [comment]
I just retried this and works for me under Chrome and Firefox on my Macbook Pro. I'm sure there is a problem - one too many divs maybe with the scrolls on the wrong div or something - I just cannot exercise the problem. Maybe we can work through it Tuesday? I'm pretty stumped.

**Comment by martenson** [comment]
@jmchilton sure, we can explore whenever you are around.

**Comment by jmchilton** [comment]
Spent some time on this today but didn't make a lot of progress:

- @mvdbeek verified this was a problem, but it sometimes works with the trackpad but not his mouse. It will scroll to the right but not the left with his trackpad.
- I've started to try to replicate the bug inside a fiddle - if anyone wants to play with it http://jsfiddle.net/Lauwkuus/5/ or http://jsfiddle.net/Lauwkuus/3/.. It was broken on my laptop for a while, but then I restarted the computer and all my problems went away.
- I've also been playing around with handsontable 3.0.0 in a local branch - but since I cannot consistently replicate the problem yet I can't tell if it is improving anything. The PR that is interesting over there is https://github.com/handsontable/handsontable/pull/4994.
